Simple-Projects/Simple_Projects/Basic/Complex/Complex.cpp

```diff
@@ -19,7 +19,7 @@ bool Complex::operator==(const Complex& other) const {
 }
 
 bool Complex::operator!=(const Complex& other) const {
-    return real != other.real || imaginary != other.imaginary;
+    return !(*this == other);
```

Comment:
Why not depend on the equality? The optimizer will certainly inline the code called, so that it will not be a pessimization